test/streaming_engine_integration.js

```diff
@@ -522,6 +522,7 @@ describe('StreamingEngine', function() {
    * Initializes or switches to the given period.
    *
    * @param {shakaExtern.Period} period
+   * @return {Object}
```

Comment:
I changed to `@return {{audio: string, video: string}}`.

As you know the shaka code better than I can you suggest a tighter type?